recipes/matlab-runtime-9.7/recipe.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: matlab-runtime-9.7
+  version: 9
```

Comment:
I would have used 9.7 ou 97 here but this is rather useless since the version is in the package name.

Reply:
It's release 9 of version 9.7. I wanted to avoid `matlab-runtime-9.7-9.7.9`, `matlab-runtime-9.7-9` make more sense.